src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1070,20 +1088,48 @@ private void StartDatabaseOnTimer(string name, DateTime? wakeup = null)
                     if (_serverStore.ServerShutdown.IsCancellationRequested)
                         return;
 
-                    _ = TryGetOrCreateResourceStore(name, wakeup).ContinueWith(t =>
-                          {
-                              var ex = t.Exception.ExtractSingleInnerException();
-                              if (ex is DatabaseConcurrentLoadTimeoutException e)
-                              {
-                                  // database failed to load, retry after 1 min
-                                  ForTestingPurposes?.RescheduleDatabaseWakeupMre?.Set();
+                    switch (nextIdleDatabaseActivity.IdleDatabaseActivityType)
+                    {
+                        case IdleDatabaseActivityType.UpdateBackupStatusOnly:
+                            var backupStatus = BackupUtils.GetBackupStatusFromCluster(_serverStore, databaseName, nextIdleDatabaseActivity.TaskId);
+                            
+                            backupStatus.LastIncrementalBackup = backupStatus.LastIncrementalBackupInternal = nextIdleDatabaseActivity.TimeOfActivityUtc;
+                            backupStatus.LocalBackup.LastIncrementalBackup = nextIdleDatabaseActivity.TimeOfActivityUtc;
+                            backupStatus.LocalBackup.IncrementalBackupDurationInMs = 0;
+
+                            var backupResult = new BackupResult();
+                            backupResult.AddMessage($"Skipping incremental backup because no changes were made from last full backup on {backupStatus.LastFullBackup}.");
+
+                            BackupUtils.SaveBackupStatus(backupStatus, databaseName, _serverStore, _logger, backupResult);
```

Comment:
What happens if we weren't able to save the backup status?
When will it wake up again?

Reply:
We swallow all exceptions within the `BackupUtils.SaveBackupStatus` method.

Therefore, the time of the next awakening will be determined within the `BackupUtils.GetEarliestIdleDatabaseActivity` method and will coincide with the time of the next backup.